arcgis-ios-sdk-samples/Edit data/Add features with contingent values/AddFeaturesContingentValuesViewController.swift

```diff
@@ -0,0 +1,202 @@
+            presentAlert(error: error)
+        }
+    }
+    
+    /// Creates a map with a vector tiled layer basemap.
+    func makeMap() -> AGSMap {
+        // Get the vector tiled layer by name.
+        let fillmoreVectorTiledLayer = AGSArcGISVectorTiledLayer(name: "FillmoreTopographicMap")
+        // Use the vector tiled layer as a basemap.
+        let basemap = AGSBasemap(baseLayer: fillmoreVectorTiledLayer)
+        let map = AGSMap(basemap: basemap)
+        return map
+    }
+    
+    /// Add a single feature to the map.
+    func addFeature(at mapPoint: AGSPoint) {
+        // Create a symbol to represent a bird's nest.
+        let symbol = AGSSimpleMarkerSymbol(style: .circle , color: .black, size: 11)
+        // Add the graphic to the graphics overlay.
+        self.graphicsOverlay.graphics.add(AGSGraphic(geometry: mapPoint, symbol: symbol))
+        // Show the attributes table view.
+        performSegue(withIdentifier: "AddFeature", sender: self)
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Set the map view's touch delegate.
+        mapView.touchDelegate = self
+        
+        // Add the source code button item to the right of navigation bar.
+        (navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = [
+            "AddFeaturesContingentValuesViewController",
+            "ContingentValuesTableViewController"
+        ]
+    }
+    
+    // MARK: - Navigation
+    
+    /// Prepare for the segue.
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        // Set the controller and its properties.
+        if let navigationController = segue.destination as? UINavigationController,
+           let controller = navigationController.viewControllers.first as? ContingentValuesTableViewController {
+            controller.isModalInPresentation = true
+            controller.featureTable = featureTable
+            controller.graphicsOverlay = graphicsOverlay
+            controller.mapPoint = mapPoint
+            controller.delegate = self
+        }
+    }
+}
+
+extension AddFeaturesContingentValuesViewController: ContingentValuesDelegate {
+    /// Create buffer graphics for the features.
+    func createBufferGraphics() {
+        // Create the query parameters.
+        let queryParameters = AGSQueryParameters()
+        // Set the where clause to filter for buffer sizes greater than 0.
+        queryParameters.whereClause = "BufferSize > 0"
+        // Create an array of graphics to add to the graphics overlay.
+        var graphics = [AGSGraphic]()
+        // Create a graphics overlay.
+        let graphicsOverlay = AGSGraphicsOverlay()
+        // Create the outline for the buffers.
+        let lineSymbol = AGSSimpleLineSymbol(style: .solid, color: .black, width: 2)
+        // Create the buffer symbol.
+        let bufferSymbol = AGSSimpleFillSymbol(style: .forwardDiagonal, color: .red, outline: lineSymbol)
```

Comment:
Similar to a comment above, can you create a single graphics overlay for the buffer graphics, set a renderer with outlined polygon symbol, and use it here?

Reply:
This is actually not possible since each feature has a different buffer size. More info from Preeti on the Symbology team: 

> Unfortunately we don't have a mechanism to define symbols based on attribute coming from data source. In ArcGIS system this is done via VisualVariables and we currently don't have it.